tests/test_arr_venue_v2.py

```diff
@@ -1478,13 +1504,17 @@ def test_unavailability_process_functions(self, client, openreview_client, helpe
             )
         )
 
+        helpers.await_queue_edit(openreview_client, edit_id=ethics_reviewer_note_edit['id'])
         helpers.await_queue_edit(openreview_client, edit_id=reviewer_note_edit['id'])
         helpers.await_queue_edit(openreview_client, edit_id=ac_note_edit['id'])
         helpers.await_queue_edit(openreview_client, edit_id=sac_note_edit['id'])
 
+        august_ethics_reviewer_edges = {o['id']['tail']: [j['weight'] for j in o['values']] for o in pc_client_v2.get_grouped_edges(invitation=f"{august_venue.get_ethics_reviewers_id()}/-/Custom_Max_Papers", groupby='tail', select='weight')}
```

Comment:
where do you plan to use this edge? is there a matching for ethics reviewers?

Reply:
Oh yeah I have to put an edge browser link in the ethics chairs webfield to show the edges

Reply:
can you also push the edge browser link change in the webfield too?